internal/setup/setup_test.go

```diff
@@ -1313,37 +1312,20 @@ var setupTests = []setupTest{{
 }}
 
 var defaultChiselYaml = `
-	format: chisel-v1
+	format: v1
 	archives:
 		ubuntu:
 			version: 22.04
 			components: [main, universe]
-			v1-public-keys: [test-key]
-	v1-public-keys:
+			public-keys: [test-key]
+	public-keys:
 		test-key:
 			id: ` + testKey.ID + `
 			armor: |` + "\n" + testutil.PrefixEachLine(testKey.PubKeyArmor, "\t\t\t\t\t\t") + `
 `
 
 func (s *S) TestParseRelease(c *C) {
-	// Run tests for format chisel-v1.
 	runParseReleaseTests(c, setupTests)
```

Comment:
I think now we can move `runParseReleaseTests` into `TestParseRelease` and avoid having two functions.

Reply:
Done in https://github.com/letFunny/chisel/pull/16/commits/ecf93be81e5c7acacb6493cbd2ae83b032ea89b8.